custom_components/ryanair/translations/pt.json

```diff
@@ -0,0 +1,30 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Equipamento já configurado"
```

Comment:
According to Google this  translate to "Equipment already configured". If that's the case, is "Dispositivo já configurado" a truer translation?

Reply:
Yes

Reply:
In that case could you update this?